bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/BigtableTableAdminGrpcClient.java

```diff
@@ -50,17 +52,14 @@ public static BigtableTableAdminClient createClient(
     return new BigtableTableAdminGrpcClient(channel);
   }
 
-  protected final CloseableChannel channel;
   private final BigtableTableServiceGrpc.BigtableTableServiceBlockingStub blockingStub;
 
   /**
    * Create a new BigtableTableAdminGrpcClient. When constructed, the client takes ownership of the
```

Comment:
These comments are no longer correct (the client no longer takes ownership of the Channel)